console/ping.go

```diff
@@ -48,21 +49,21 @@ func (self *CmdApierPing) Name() string {
 
 func (self *CmdApierPing) RpcMethod() string {
 	switch self.rpcParams.Item {
```

Comment:
Please make the Item to lower and not the suppliers since that is a constant and we don't need to loose processing on run time to make it low.